content/en/docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -94,21 +100,19 @@ events {
     worker_connections  1024;
 ```
 
-
-
 <!-- discussion -->
 
-## Understanding Process Namespace Sharing
+## Understanding process namespace sharing
```

Comment:
Is this a concept that should be capitalized?

Reply:
I do not have an idea how we are supposed to format "concepts". Sometimes will use italics (which may not be appropriate for all localizations), sometimes we use quotes (single or double), other times we capitalize the first character of each word ...

Reply:
In running English text, we [use italics to introduce new terms](https://kubernetes.io/docs/contribute/style/style-guide/#use-italics-to-define-or-introduce-new-terms). Localization teams are free to pick an appropriate way to emphasize (or not emphasize) new terms; this could be formatting, or it could be a rewording.

[Headings should be sentence case](https://kubernetes.io/docs/contribute/style/style-guide/#headings). If we would not write “Process Namespace Sharing” with initial capitals in running text, we should not capitalize it in a heading either.